mmdet/core/hook/sync_random_size_hook.py

```diff
@@ -33,9 +35,9 @@ def __init__(self,
         self.interval = interval
         self.device = device
 
-    def after_train_iter(self, runner):
+    def after_train_epoch(self, runner):
```

Comment:
Please add some notes: its behavior is different from YOLOX's official implementation, the official is to change the size every fixed iteration interval, and we write it to change the size every fixed epoch. And the reason for the modification is that under the current design, the dataloader multi-process cannot modify the size in real-time.